spec/licensee/commands/detect_spec.rb

```diff
@@ -72,7 +72,7 @@
     it 'returns the expected output' do
       msg = +'`licensee detect --json` output did not match expectations. '
       msg << 'Run `script/dump-detect-json-fixture` and verify the output.'
-      expect(JSON.parse(stdout)).to eql(expected), msg
+      expect(JSON.parse(stdout).tap { |h| h['matched_files'][1].delete('content') }).to eql(expected), msg
```

Comment:
Yes the same `.tap` call again. This isn't required anywhere else so duplication strikes me as better than a helper method elsewhere. Happy to be wrong!